string target_identifier
int32 target_index
bool restart
---
bool success
string message
